cumulusci/core/config/project_config.py

```diff
@@ -421,9 +433,9 @@ def get_previous_version(self):
     def config_project_path(self):
         if not self.repo_root:
             return
-        path = os.path.join(self.repo_root, self.config_filename)
-        if os.path.isfile(path):
-            return path
+        path = Path(self.repo_root) / self.config_filename
+        if path.is_file():
+            return str(path)
```

Comment:
What's your logic for when to convert a Path to a str and when to leave it as a Path? You're doing the former here but the latter in config_project_local_path.

Reply:
good catch, let me make both of them return a string. I think it's a little safer than returning a `Path` object.